Formula/dvc.rb

```diff
@@ -3,15 +3,10 @@ class Dvc < Formula
 
   desc "Git for data science projects"
   homepage "https://dvc.org"
-  url "https://github.com/iterative/dvc/archive/1.9.1.tar.gz"
-  sha256 "9488698a341719a5ea2baf3635871715c83c7b72081f81119ece8ce2b0a54ffd"
+  url "https://files.pythonhosted.org/packages/df/6c/4f1751894f195a341eae470152fdf73c0749ebc3f158774831142e3d54ab/dvc-1.10.0.tar.gz"
+  sha256 "ceaeab6ac7c0b36d0d9cbc638f3d5a09446ea140a823c04aafbb6203da7ee727"
   license "Apache-2.0"
 
-  livecheck do
-    url "https://github.com/iterative/dvc.git"
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
-  end
-
```

Comment:
Why are you removing `livecheck` block?

Reply:
I changed the url to point to [pypi](https://pypi.org/) instead as it this was recommended to me once as a better way to get the formulas for upgrading the resources blocks in the future. Hence I don't need that block since the default `livecheck` works for this url. I made sure to test this locally on the branch.
```sh
$ brew livecheck dvc
dvc : 1.10.0 ==> 1.10.0
```

Reply:
This works fine, but we should keep the block to ensure that `livecheck` won't break in the future if a `head` URL is added:

```ruby
livecheck do
  url :stable
end
```

Reply:
@Rylan12 I could add that. I am curious then about 2 things then:
1. What is the `head` URL for?
2. I have seen some formulas with no `livecheck` block and I assumed because there were using one of the [strategies](https://github.com/Homebrew/brew/tree/master/Library/Homebrew/livecheck/strategy) implemented. SO my question is that if I encounter a missing `livecheck` block for an updated formula, should i add the same one that you proposed on here?

Reply:
An example would be PR https://github.com/Homebrew/homebrew-core/pull/65248/files where I assume the `git` `livecheck` is performed but I don't see a block for it.

Reply:
> 1. What is the `head` URL for?

Take a look at the [`head` documentation](https://docs.brew.sh/Formula-Cookbook#unstable-versions-head) for some more info. Essentially, it allows users to install the most recent development version of a formula. It's optional and not on all formulae.

> 2. I have seen some formulas with no `livecheck` block and I assumed because there were using one of the [strategies](https://github.com/Homebrew/brew/tree/master/Library/Homebrew/livecheck/strategy) implemented. SO my question is that if I encounter a missing `livecheck` block for an updated formula, should i add the same one that you proposed on here?

`livecheck` can still run on formulae that don't specify a livecheck block (as you pointed out). When no block is specified, `livecheck` attempts to find version of the available URLs in the following order: `head`, `stable` (which is the "main" URL), `homepage`. In this case, there's no `head` URL so it defaults to the `stable` URL (which is equivalent to specifying `url :stable` in the `livecheck` block). However, if a `head` URL is added in the future, `livecheck` would switch to checking the `head` URL if we don't specify `url :stable`. So, essentially, adding the `livecheck` block is just a precaution (arguably not necessary).

In the PR you linked, this likely won't be an issue because if a `head` spec is added it will likely be for the same `git` URL that's specified in the `stable` spec. We're still in the process of refining `livecheck` functionality and the `livecheck` blocks so things aren't totally consistent across the board, unfortunately.